src/ui/Button/Button.stories.tsx

```diff
@@ -1,38 +1,51 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { withDesign } from 'storybook-addon-designs';
 
-import { Box } from '../Box/Box';
+import { Flex, Text, Box, TextArea, HR } from 'ui';
 
 import { Button, ButtonStory } from './Button';
 
 export default {
-  title: 'Design System/Components/Button',
```

Comment:
These titles to make the component's menu one less level deep, what exactly are we benefiting from removing this?